### PR TITLE
Fix: stash items

### LINKED ISSUE #172: Stash
Stash will crash client on stowing multiple items like golden tokens, silver tokens, many new stackable items like id 36190, 36191, 36189, 36171. Those are few examples.

I wrote a converter to make potions, ammo, valueable, runes and creatureproducts stowable only and made new items.xml. It will take care of like 95% of crashable items. It's a temp fix.

[items.zip](https://github.com/opentibiabr/otservbr-global/files/5136042/items.zip)

### LINKED ISSUE #1692: Stash
Stash will crash client on stowing multiple items like golden tokens, silver tokens, many new stackable items like id 36190, 36191, 36189, 36171. Those are few examples.

I wrote a converter to make potions, ammo, valueable, runes and creatureproducts stowable only and made new items.xml. It will take care of like 95% of crashable items. It's a temp fix.

[items.zip](https://github.com/opentibiabr/otservbr-global/files/5136042/items.zip)

### PR DESCRIPTION
**Pull request by marcosvf132**
Resolves #1692
A full review on all stowable items.

### CODE REVIEW

File: data/items/items.xml [hunk 1]
```diff
@@ -12058,7 +12059,6 @@
 		<attribute key="weight" value="3000" />
 	</item>
 	<item id="6536" name="Countess Sorrow's frozen tear">
-		<attribute key="loottype" value="valuable" />
```
Comment (marcosvf132) [inline data/items/items.xml:12061]: Two loottype

File: data/items/items.xml [hunk 2]
```diff
@@ -12076,7 +12076,6 @@
 		<attribute key="weight" value="3500" />
 	</item>
 	<item id="6540" article="a" name="piece of Massacre's shell">
-		<attribute key="loottype" value="creatureproduct" />
```
Comment (marcosvf132) [inline data/items/items.xml:12079]: Same here, two loot types.

File: data/items/items.xml [hunk 3]
```diff
@@ -42505,8 +42539,7 @@
 	<item id="32142" article="an" name="exercise dummy">
 		<attribute key="description" value="You can train your skills by using training weapons with this exercise dummy." />
 	</item>
-	<item id="32143" article="a" name="
-	exercise dummy">
```
Comment (marcosvf132) [inline data/items/items.xml:42509]: Typo

**Comment by dudantas** [comment]
Is finished?

**Review by dudantas — APPROVED** [review]

**Review by majestyotbr — APPROVED** [review]

**Pull request merged**